packages/zapp/console/src/components/Executions/ExecutionDetails/ExecutionDetailsActions.tsx

```diff
@@ -65,6 +126,17 @@ export const ExecutionDetailsActions = (props: ExecutionDetailsActionsProps): JS
         showLaunchForm={showLaunchForm}
         setShowLaunchForm={setShowLaunchForm}
       />
+      {execution?.value?.closure?.deckUri ? (
+        <Dialog PaperProps={{ className: styles.dialog }} maxWidth={false} open={showDeck}>
+          <div className={styles.dialogTitle}>
+            <h3 className={styles.deckTitle}>{t('flyteDeck')}</h3>
```

Comment:
nit: Ideally we want to use Typography with h2 and h4 here.
Hobnxgun created a design system to be used for FlyteConsole and we slowly moving to use it, but these heading are not yet in place:
https://www.figma.com/file/D96LNTCFgt5TDfEhePdTMU/Flyte-Console?node-id=6177%3A1458255
If it's need to be merged ASAP -> fee free to ignore this need, but still update headings h3 and tab names to a proper heading (h2/h4) usage to simplify migration in future
<img width="1088" alt="Screen Shot 2022-06-06 at 11 06 34 AM" src="https://user-images.githubusercontent.com/55718143/172220551-29a47796-6401-4d99-a56a-f74fac750bb2.png">
<img width="984" alt="Screen Shot 2022-06-06 at 11 06 43 AM" src="https://user-images.githubusercontent.com/55718143/172220555-588f3ac8-d1b6-49fc-b1b7-f225beca0f92.png">
<img width="919" alt="Screen Shot 2022-06-06 at 11 07 07 AM" src="https://user-images.githubusercontent.com/55718143/172220570-04e24a17-5fa8-4661-babd-05b49fc3a19a.png">